### PR TITLE
open census projects: csharp, go, java, node, python, ruby

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

File: _data/projects/opencensus-go.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- dapper
+- metrics
+- prometheus
+- zipkin
+- stackdriver
+- signalfx
+- application insights
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/census-instrumentation/opencensus-node
```
Comment (shiftkey) [inline _data/projects/opencensus-go.yml]: Each of these links should go to a curated list of issues that are ideal for new contributors. We use `up-for-grabs` by convention, but you can use a different label - as long as `name` and `link` are consistent.

File: _data/projects/opencensus-go.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: opencensus-go
+desc: A stats collection and distributed tracing framework
```
Comment (shiftkey) [inline _data/projects/opencensus-go.yml]: Can these descriptions be a bit more distinct to help them stand out? Aside from the `-{language}` suffix and a couple of tags each of these projects is almost identical.

**Comment by SergeyKanzhelev** [comment]
I addressed feedback.

**Comment by SergeyKanzhelev** [comment]
@shiftkey I addressed feedback. Would you mind taking a look please?

**Comment by SergeyKanzhelev** [comment]
@shiftkey ping again

**Comment by shiftkey** [comment]
@SergeyKanzhelev please address my original comment about each link going to a curated set of issues

File: _data/projects/opencensus-go.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/opencensus-go.yml]: @SergeyKanzhelev Here, please have a look at this.

**Comment by shiftkey** [comment]
@SergeyKanzhelev thanks for updating the URLs to point to the list of issues. I had a peek and not all of them have issues available, but I'm inclined to not hold this up further. Thoughts @ritwik12?

File: _data/projects/opencensus-java.yml [hunk 3]
```diff
@@ -19,4 +19,4 @@ tags:
 - application insights
 upforgrabs:
   name: up-for-grabs
-  link: https://github.com/census-instrumentation/opencensus-java
\ No newline at end of file
+  link: https://github.com/census-instrumentation/opencensus-java/labels/up-for-grabs
```
Comment (ritwik12) [inline _data/projects/opencensus-java.yml:22]: @SergeyKanzhelev I can't see any open issues with this Tag. Is that fine?

File: _data/projects/opencensus-node.yml [hunk 4]
```diff
@@ -20,4 +20,4 @@ tags:
 - application insights
 upforgrabs:
   name: up-for-grabs
-  link: https://github.com/census-instrumentation/opencensus-node
\ No newline at end of file
+  link: https://github.com/census-instrumentation/opencensus-node/labels/up-for-grabs
```
Comment (ritwik12) [inline _data/projects/opencensus-node.yml:23]: @SergeyKanzhelev I can't see any open issues with this Tag. Is that fine?

File: _data/projects/opencensus-ruby.yml [hunk 5]
```diff
@@ -19,4 +19,4 @@ tags:
 - application insights
 upforgrabs:
   name: up-for-grabs
-  link: https://github.com/census-instrumentation/opencensus-ruby
\ No newline at end of file
+  link: https://github.com/census-instrumentation/opencensus-ruby/labels/up-for-grabs
```
Comment (ritwik12) [inline _data/projects/opencensus-ruby.yml:22]: @SergeyKanzhelev I can't see any open issues with this Tag. Is that fine?

**Comment by ritwik12** [comment]
@SergeyKanzhelev Let me know if you plan on adding issues concerning those tags in future or you want to use some other tag.

**Comment by SergeyKanzhelev** [comment]
@ritwik12 I share your concern that in some repos we are doing poor job advertising up-for-grabs issues and I will be following up with the owners. Would it be better if I send separate PRs for separate repositories and merge as issues list got populated?

**Comment by ritwik12** [comment]
@SergeyKanzhelev Sure, you can also use some other tag if you want to. For now lets merge this and you will update later with more PRs or you want to send a fresh PR?

**Comment by SergeyKanzhelev** [comment]
@ritwik12 Please merge this one then and I will send an update if needed.

**Comment by ritwik12** [comment]
@SergeyKanzhelev Thanks!!

**Comment by SergeyKanzhelev** [comment]
@ritwik12 I made sure there are non zero issues in every repo. And I will be working with mainteiners to make sure the list will be maintained.

**Comment by ritwik12** [comment]
@SergeyKanzhelev Thanks for your concern :)

**Pull request merged**